packages/app/i18n/en.pot

```diff
@@ -402,6 +402,12 @@ msgstr "Extreme line % detection"
 msgid "Number"
 msgstr "Number"
 
+msgid "Fixed column headers to top of table"
```

Comment:
Is this the final phrasing for these options, verified by @cooper-joe ?

Reply:
No, it should be `Fix column headers...` and `Fix row headers...`.

Reply:
Nice catch, fixed!